bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -1236,6 +1238,8 @@ spec:
     name: velero-plugin-for-gcp
   - image: quay.io/konveyor/kubevirt-velero-plugin:v0.7.0
     name: kubevirt-velero-plugin
+  - image: quay.io/hypershift/hypershift-oadp-plugin:latest
```

Comment:
Your plugin repo name is `hypershift-oadp-plugin` , so as to be consistent shouldn't the name be `hypershift-oadp-plugin` too ?